library/std/src/sys_common/wtf8.rs

```diff
@@ -541,7 +541,7 @@ impl fmt::Debug for Wtf8 {
             write_str_escaped(formatter, unsafe {
                 str::from_utf8_unchecked(&self.bytes[pos..surrogate_pos])
             })?;
-            write!(formatter, "\\u{{{:x}}}", surrogate)?;
+            write!(formatter, "\\u{{{surrogate:x}}}")?;
```

Comment:
This one is also annoying to figure out because it is actually `{{}}` to make sure the `{}` tokens actually appear, then a "format arg place". It's easier to miss that subtlety this way and mistake the code as saying to write literally `{surrogate:x}`.

Reply:
reverted